EIPS/eip-ondemand-data.md

```diff
@@ -0,0 +1,152 @@
+---
```

Comment:
```suggestion
---
eip: 7412
```
Assigning EIP number using PR number.  Please also update the filename

Reply:
All set, thanks!